### PR TITLE
Add github actions CI

### PR DESCRIPTION
**Pull request by jjyao**
Add a simple CI to check the format and lint of the code. Will have a follow-up PR for running tests.

https://github.com/ray-project/ray_beam_runner/issues/5

### CODE REVIEW

**Comment by jjyao** [comment]
cc @pabloem

File: setup.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
 from setuptools import find_packages, setup
 
 setup(
-    name="ray_beam_runner",
+    name="ray_beam",
     packages=find_packages(where=".", include="ray_beam_runner*"),
     version="0.0.1",
     author="Ray Team",
     description="A Ray runner for Apache Beam",
```
Comment (pdames) [inline setup.py:8]:
```suggestion
    description="An Apache Beam Runner for Ray.",
```

File: setup.py [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
     long_description="A distributed runner for Apache Beam built on top of "
     "distributed computing framework Ray.",
```
Comment (pdames) [inline setup.py:10]:
```suggestion
    long_description="An Apache Beam Runner based on the Ray "
    "distributed computing framework.",
```

File: setup.py [hunk 3]
```diff
@@ -1,14 +1,13 @@
...
     url="https://github.com/ray-project/ray_beam_runner",
-    install_requires=[
-        "ray", "apache_beam"
-    ])
+    install_requires=["ray", "apache_beam"],
+)
```
Comment (pdames) [inline setup.py:13]: I think we should also target supporting the same Python versions as Ray:
```suggestion
      classifiers=[
          'Programming Language :: Python :: 3.6',
          'Programming Language :: Python :: 3.7',
          'Programming Language :: Python :: 3.8',
          'Programming Language :: Python :: 3.9',
      ],
)
```

**Review by pdames — APPROVED** [review]
Very cool to see our first working CI workflows, thanks! Mostly minor comments for now - feel free to discuss/commit/reject suggestions and merge.

File: setup.py (hunk 3 above)
Comment (jjyao) [reply setup.py:13]: Added excepting 3.6 since it's deprecated.

File: setup.py (hunk 1 above)
Comment (jjyao) [reply setup.py:8]: Changed to `An Apache Beam Runner using Ray.`

**Pull request merged**